pkg/sql/pgwire/testdata/pgtest/param_status

```diff
@@ -20,11 +20,9 @@ Query {"String": "SET TIME ZONE +6"}
 until crdb_only
 ParameterStatus
 ----
-{"Type":"ParameterStatus","Name":"TimeZone","Value":"6"}
+{"Type":"ParameterStatus","Name":"TimeZone","Value":"\u003c+06\u003e-06"}
 
 
-# TODO(#42404): postgres has a different output. This is what we have right now
-# as code, but we need to dig into what/why we use this format in param status.
```

Comment:
delete this stanza, and remove the `crdb_only` above

Reply:
Done